data/data-pipeline/data_pipeline/ipython/scoring_comparison.ipynb

```diff
@@ -476,6 +504,12 @@
     "            priority_communities_field=field_names.ENERGY_RELATED_COMMUNITIES_DEFINITION_ALTERNATIVE,\n",
     "            other_census_tract_fields_to_keep=[],\n",
     "        ),\n",
+    "        \n",
+    "        Index(\n",
+    "            method_name=\"CDC SVI Index\",\n",
+    "            priority_communities_field=field_names.CDC_SVI_INDEX_THEMES_PRIORITY_COMMUNITY,\n",
+    "            other_census_tract_fields_to_keep=RELEVANT_CDC_SVI_INDEX_COLUMNS,\n",
```

Comment:
My fault but FYI this field doesn't do anything any more, it's legacy / deprecated code: https://github.com/usds/justice40-tool/issues/1129

Reply:
Done